ext/scheduler/airflow/dag/expected_dag.2.1.py

```diff
@@ -102,7 +102,6 @@ def get_entrypoint_cmd(plugin_entrypoint_script):
     k8s.V1EnvVar(name="JOB_NAME", value='infra.billing.weekly-status-reports'),
     k8s.V1EnvVar(name="OPTIMUS_HOST", value='http://optimus.example.com'),
     k8s.V1EnvVar(name="PROJECT", value='example-proj'),
-    k8s.V1EnvVar(name="SCHEDULED_AT", value='{{ next_execution_date }}'),
```

Comment:
this needs to be replicated to 2.4 and 2.6 templates !

Reply:
they are using `data_interval_end`

Reply:
ohh okay, then ill remove this

Reply:
have removed this from 2.4 and 2.6 as well, can you check